### PR TITLE
Added activate action

### PR DESCRIPTION
**Pull request by abulrim**
- Added request mocker to mock responses in tests
- Added decorators to handle complex view logic
- Added action tests
- Updated conduit version

### CODE REVIEW

**Comment by abulrim** [comment]
Still needs to be tested against connect, opened PR to get feedback

**Comment by pedromartinez** [comment]
@abulrim Please post a link to this PR in Slack, #connect-ruby, and the team will review it.

**Comment by captainbeardo** [comment]
👍

**Comment by dougperez** [comment]
Need a version update?

**Comment by abulrim** [comment]
updated version to `0.4.5`

File: lib/conduit/atom/driver.rb [hunk 1]
```diff
@@ -9,6 +9,9 @@ module Atom
       required_credentials :token
       optional_attributes  :mock_status, :host_override
 
+      # Create and activate a line
+      action :line_activation
```
Comment (dougperez) [inline lib/conduit/atom/driver.rb]: I know for other conduit drivers we call this just ```activate``` not sure if it matters here or you actually want this named ```line_activation ```, but just an FYI. 

Examples;
https://github.com/conduit/conduit-sprint/blob/master/lib/conduit/sprint/driver.rb#L9
https://github.com/conduit/conduit-cogotech/blob/master/lib/conduit/cogotech/driver.rb#L9
https://github.com/conduit/conduit-fusion/blob/master/lib/conduit/fusion/driver.rb#L13

**Comment by dougperez** [comment]
Just has the one comment left otherwise. 👍

**Comment by abulrim** [comment]
Hm... yeah I think it's better to stay consistent. I'll change to `activate`.

**Comment by abulrim** [comment]
Renamed to `activate`, bumped conduit version

**Pull request merged**